contracts/sophia.md

```diff
@@ -453,7 +468,7 @@ The block-chain environment available to a contract is defined in three name spa
   calling the contract.
 - `Call.value` is the amount of coins transferred to the contract in the call.
 - `Call.gas_price` is the gas price of the current call.
-- `Call.gas_left` is the amount of gas left for the current call.
+- `Call.gas_left()` is the amount of gas left for the current call.
 - `Chain.get_balance(a : address)` returns the balance of account `a`.
```

Comment:
It feels a bit strange for the reader that `Chain.spend` is in namespace Chain though is not listed outside of this section. Though I do not have better proposal so ok.